csharp/ql/src/semmle/code/csharp/Namespace.qll

```diff
@@ -18,7 +18,7 @@ class TypeContainer extends DotNet::NamedElement, Element, @type_container { }
  * }
  * ```
  */
-class Namespace extends DotNet::Namespace, TypeContainer, @namespace {
+class Namespace extends DotNet::Namespace, TypeContainer, Declaration, @namespace {
```

Comment:
Should we add `DotNet::Declaration` as a base class of `DotNet::Namespace` as well?

Reply:
Come to think of it, we should run a dist-compare to check that performance is unchanged. (Changing the type hierarchy has resulted in unexpected optimizer behaviour before.)